### ISSUE TITLE
Scylla setup fails on Amazon2 - cpupower.service not found

### ISSUE THREAD

**Issue by nizarh11** [scylladb/scylla#6882]
*Installation details*
Scylla version (or git commit hash): 666.development-0.20200720.5371be71e90
Cluster size:1
OS (RHEL/CentOS/Ubuntu/AWS AMI): ami-099a8245f5daa82bf
Test Id: 5bdd2564-3574-4590-bae2-c7879d05eb3d
Scylla repo: https://s3.amazonaws.com/downloads.scylladb.com/rpm/unstable/centos/master/2020-07-20T01:39:33Z/scylla.repo

**scylla_util.SystemdException: unit cpupower.service is not found or invalid**

```
Command: 'sudo /usr/lib/scylla/scylla_setup --nic eth0 --disks /dev/nvme0n1 '

Exit code: 1

Stdout:

tuning: /sys/devices/pci0000:00/0000:00:1e.0/nvme/nvme0/nvme0n1/queue/nomerges 2
tuning /sys/devices/pci0000:00/0000:00:1e.0/nvme/nvme0/nvme0n1
tuning /sys/devices/pci0000:00/0000:00:1e.0/nvme/nvme0/nvme0n1
tuning /sys/devices/pci0000:00/0000:00:1e.0/nvme/nvme0/nvme0n1
tuning /sys/devices/pci0000:00/0000:00:1e.0/nvme/nvme0/nvme0n1
node_exporter successfully installed
Loaded plugins: extras_suggestions, langpacks, priorities, update-motd
Package kernel-tools-4.14.181-142.260.amzn2.x86_64 already installed and latest version
Nothing to do
CPU scaling setup failed.

Stderr:

subprocess.CalledProcessError: Command '['systemctl', 'cat', 'cpupower.service']' returned non-zero exit status 1.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/scylladb/scripts/libexec/scylla_cpuscaling_setup", line 87, in <module>
    cpupwr = systemd_unit('cpupower.service')
  File "/opt/scylladb/scripts/scylla_util.py", line 632, in __init__
    raise SystemdException('unit {} is not found or invalid'.format(unit))
scylla_util.SystemdException: unit cpupower.service is not found or invalid


```

Logs:

```
+------------------------------------------------------------------------------------------------------------------------------------------------------------+
|                                          Log links for testrun with test id 5bdd2564-3574-4590-bae2-c7879d05eb3d                                           |
+-----------------+------------+-----------------------------------------------------------------------------------------------------------------------------+
| Date            | Log type   | Link                                                                                                                        |
+-----------------+------------+-----------------------------------------------------------------------------------------------------------------------------+
| 20200720_092637 | db-cluster | https://cloudius-jenkins-test.s3.amazonaws.com/5bdd2564-3574-4590-bae2-c7879d05eb3d/20200720_092637/db-cluster-5bdd2564.zip |
| 20200720_092637 | sct-runner | https://cloudius-jenkins-test.s3.amazonaws.com/5bdd2564-3574-4590-bae2-c7879d05eb3d/20200720_092637/sct-runner-5bdd2564.zip |
+-----------------+------------+-----------------------------------------------------------------------------------------------------------------------------+

```

**Comment by nizarh11** [comment]
I see the issue was previously fixed on https://github.com/scylladb/scylla/pull/6621

**Comment by slivne** [comment]
@tzach please note we will not support Amazon Linux2 in 4.2

**Comment by penberg** [comment]
@slivne We could backport commit 06bcbfc4c3a157d754cac02536aaca1e966d13ea to 4.2 and 4.1. The fix is non-intrusive and only affects Amazon Linux 2.

**Comment by penberg** [comment]
@nizarh11 Yeah, it was fixed, and as you are running 5371be71e90, you should have the fix. @syuu1228 any ideas what's going on here?

**Comment by syuu1228** [comment]
Here's a reason why it's failing.. the script fails to detect Amazon Linux 2.

Previously we realized that `distro.id()` is NOT always same output as ID in /etc/os-release, we modified the value based on [their document](https://distro.readthedocs.io/en/latest/).
see commit: ad87ffc60d6816c2101d4453dd028099099f0815

Even though, on Amazon linux 2, `distro.id()` does NOT returns "amazon", as document described.
It actually SAME as /etc/os-release, "amzn":
```
[ec2-user@ip-10-0-0-10 scripts]$ sudo /opt/scylladb/python3/bin/python3
Python 3.8.3 (default, May 15 2020, 00:00:00) 
[GCC 10.1.1 20200507 (Red Hat 10.1.1-1)] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import distro
>>> distro.id()
'amzn'
```
Maybe their document is wrong.

I found some tests that expects Amazon Linux returns "amzn":
https://github.com/nir0s/distro/blob/bae8670e1d398180bca44bb4b8275669d74e6676/tests/test_distro.py#L1528
